### PR TITLE
Concept Anchor - Formal pattern improvements

### PR DESCRIPTION
**Pull request by fioddor**
- Move pattern status to its section
- Fill in the resulting context

### CODE REVIEW

File: patterns/1-initial/concept-anchor.md [hunk 1]
```diff
@@ -71,11 +69,13 @@ InnerSource helps in this process by its emphasis on a CI/CD workflow as well as
 
 ## Resulting Context
 
-TBD
+Management and newcomers to InnerSource engage in conversations around InnerSource, percieved benefits, expected difficulties, etc. Usually comparing to  previous experiences around the anchored concepts.
```
Comment (spier) [inline patterns/1-initial/concept-anchor.md]:
```suggestion
Management and newcomers to InnerSource engage in conversations around InnerSource, perceived benefits, expected difficulties, etc.
They are usually comparing InnerSource to previous experiences around the anchored concepts.
```

Splitting into two lines.
Fix typo.
Extend a bit.

File: patterns/1-initial/concept-anchor.md [hunk 2]
```diff
@@ -71,11 +69,13 @@ InnerSource helps in this process by its emphasis on a CI/CD workflow as well as
...
 
 ## Status
 
-* Initial (Donut)
+Initial.
+These are ideas for solutions to the problem that have not been tested yet.
```
Comment (spier) [inline patterns/1-initial/concept-anchor.md]:
```suggestion
* Initial

The above are ideas for solutions to the problem that have not been tested yet.
```

File: patterns/1-initial/concept-anchor.md [hunk 3]
```diff
@@ -71,11 +69,13 @@ InnerSource helps in this process by its emphasis on a CI/CD workflow as well as
...
+This is still a pattern any Known Instances.
```
Comment (spier) [inline patterns/1-initial/concept-anchor.md]:
```suggestion
This pattern does not have any Known Instances yet.
```

I wasn't sure what you meant here.

Tried to reword it. Did you mean that?

**Review by spier — COMMENTED** [review]
Love the incremental approach you are taking to improving the patterns!
Left some comments inline.

**Comment by fioddor** [comment]
Thanks for your review, @spier. I'm discovering that I'm multitasking too much. You are catching embarrasing obvious flaws that slip through my context juggling.

**Review by spier — COMMENTED** [review]
LGTM

**Review by spier — APPROVED** [review]

**Comment by spier** [comment]
> Thanks for your review, @spier. I'm discovering that I'm multitasking too much. You are catching embarrasing obvious flaws that slip through my context juggling.

Hehe no worries @fioddor. 

Thanks again for your contributions here!

**Comment by winterrocks** [comment]
OpenChain Project has created short Explainers (https://github.com/OpenChain-Project/Reference-Material/tree/master/Education-For-Internal-Teams) and it may be worth having a look at those and get some ideas for this InnerSource pattern. 

OpenChain Project explainers are for
-  C-suite
- Compliance Risk Legal
- Dev Team Leader
- Developers
- HR
- Procurement
- Sales / Marketing
- Training

**Comment by spier** [comment]
@winterrocks are you proposing to write similar explainers for InnerSource, and add them to the Concept Anchor pattern?

Btw how did you come across this old PR? :)

**Comment by winterrocks** [comment]
That may be one idea. I just threw the idea for a wider audience here. 

How I came across this PR, I do not remember now - pure luck...

**Pull request merged**